### ISSUE TITLE
Revisão da fila de publicação

### ISSUE THREAD

**Issue by sahrosa** [servicosgovbr/editor-de-servicos#339]

**Comment by sahrosa** [comment]
O que acham @oliviagj @mformiga @almeidafab?

![screen shot 2015-09-28 at 4 58 25 pm](https://cloud.githubusercontent.com/assets/2932644/10147090/670ee302-6602-11e5-9b91-745820c49aed.png)

**Comment by mformiga** [comment]
@sahrosa Gostei da tela. Aproveito para perguntar como estamos chamando a fila de publicação de um serviço a partir das sugestões de cidadãos, servidores de outros órgãos, sociedade em geral.

**Comment by sahrosa** [comment]
Oi @mformiga! Que bom que gostou! 

Sobre as tuas dúvidas, até conversei com o @kchnr pra confirmar, mas como vamos usar o gerid como meio de login, e não podemos permitir a edição anônima, por enquanto, teremos apenas os servidores de cada órgão, editando os serviços dos seus órgãos. Já o perfil de admin, terá acesso a todos os serviços.

No caso dos cidadãos, seria necessário uma outra forma de identifica-los e permitir que façam edições, e por isso, não estamos tratando este cenário, por ser um esforço adicional, uma vez que o gerid funciona apenas com servidores.

**Comment by mformiga** [comment]
Olá @sahrosa e @kchnr O que acontecerá se pessoas diferentes editarem o serviço ? O mesmo serviço irá aparecer na fila várias vezes misturado com todas as outras edições do órgão ou poderemos visualizar em uma fila todas as alterações sugeridas para um serviço específico ? Parece-me que a segunda opção é bem mais gerenciável. O que acham ?

**Comment by sahrosa** [comment]
Oi @mformiga! Legal teu ponto!

Todas as edições de um serviço específico aparecem uma única vez na fila, sem se repetir várias vezes :)

**Comment by mformiga** [comment]
@sahrosa esta tela já está implementada ?

**Comment by sahrosa** [comment]
Não! Queria validar a ideia antes! Vamos começar a implementar em seguida!

**Comment by sahrosa** [comment]
@kchnr @brunoleite preciso de ajuda pra colocar os botões no cabeçalho e fazer as coisas funcionarem do jeito correto. Alguém pode parear comigo na segunda?

Por enquanto está assim: 
 
![screen shot 2015-10-02 at 5 06 35 pm](https://cloud.githubusercontent.com/assets/2932644/10256491/f6a97a22-6927-11e5-94e4-37d66a4a1736.png)

**Comment by sahrosa** [comment]
Conforme combinamos, @kchnr, esse cartão fica contigo por ser mais técnico :)

**Comment by glauberramos** [comment]
Pessoal acreditamos que esse cartão está pronto, podemos passar para homologação @almeidafab @mformiga @gustavo-freitas?

**Comment by gustavo-freitas** [comment]
Claro! @glauberramos.

**Comment by gustavo-freitas** [comment]
Card validado com @almeidafab .
